docker/caddy/Dockerfile

```diff
@@ -1,20 +1,7 @@
-FROM node:12.18.0 AS client-builder
-
-COPY src ./src
-COPY static ./static
-COPY gatsby-config.js .
-COPY package.json .
-COPY yarn.lock .
-
-ENV CYPRESS_INSTALL_BINARY 0
-RUN yarn --frozen-lockfile
-RUN yarn build
-
 FROM caddy:2.1.1-builder AS caddy-builder
 
-RUN caddy-builder github.com/caddy-dns/cloudflare
+RUN caddy-builder github.com/caddy-dns/cloudflare github.com/caddy-dns/route53
```

Comment:
Do we still need `github.com/caddy-dns/cloudflare`?

Reply:
not really, leaving it there for anyone else to use for now